.github/ISSUE_TEMPLATE/weakness_report.yml

```diff
@@ -0,0 +1,92 @@
+name: Weakness Report
+description: Report a weakness in the sense of the CWE; a potential source of vulnerabilities.
+#title: ""
+labels: ["weakness"]
+#assignees:
+#  -
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: summary
+    attributes:
+      label: Summary
+      description: A clear and specific description of what the weakness is.
+    validations:
+      required: true
+  - type: checkboxes
+    id: source-of-vulnerability
+    attributes:
+      label: Security vulnerability
+      description: Would you consider this weakness a potential source of vulnerabilities?
+      options:
+        - label: Yes, and I will add the 'security' label to this issue.
+          required: false
+  - type: textarea
```

Comment:
These form fields don't really make sense here, since many such things are design issues, rather than I-did-this-and-it-did-the-wrong-thing reports.

I think we should mostly just give them a free text field here.